config/vufind/Folio.ini

```diff
@@ -116,10 +116,19 @@ defaultRequiredDate = 0:1:0
 ; "pickUpLocation", "proxiedUsers" and "requestGroup"
 extraHoldFields = requiredByDate:pickUpLocation
 
-; By default, a "Hold" type request is placed when an item is unavailable. Set to
-; "Recall" if a recall is preferred, or "Page" if a page is preferred.
+; By default, a "Hold" type request is placed when an item is unavailable and a Page
+; when an item is available. This setting overrides the default behavior for
+; unavailable items and for all title-level requests. Legal values: "Page", "Hold" or
+; "Recall"
 ;default_request = Hold
 
+; It is possible that the default_request method will not work in every situation; if
```

Comment:
"will not work in every situation" is true but implies (IMHO) a problem. rather than something more likely intentional.  Maybe add something like "such as due to specific circulation rules."

Reply:
Thanks, @maccabeelevine. Comment improved in commit e96af67887f4ff40d02ec7960ceb8f8475a937fc.